app/lib/l10n/app_en.arb

```diff
@@ -222,7 +222,16 @@
   "remove": "Remove",
   "reset": "Reset",
   "sasGotIt": "Got it",
-  "sasIncomingReqNotifContent": "%s wants to verify your session",
+  "sasIncomingReqNotifContent": "{sender} wants to verify your session",
+  "@sasIncomingReqNotifContent": {
+    "description": "A message with a single parameter",
```

Comment:
this is a bad description.
```suggestion
    "description": "When another users asks to verify this sessions",
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1539/commits/0a6fcae7725f68ece46f80a92bcfccf852c8f2c0